jbmc/src/java_bytecode/java_bytecode_concurrency_instrumentation.cpp

```diff
@@ -361,3 +497,58 @@ void convert_threadblock(symbol_tablet &symbol_table)
     }
   }
 }
+
+/// Iterate through the symbol table to find and appropriately instrument
```

Comment:
The word `appropriately` adds no information. Remove it.

Reply:
done